src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -83,6 +83,8 @@
             "alignments as well as the proportion of the reads that passed machine signal-to-noise threshold quality filters. " +
             "Note that these quality filters are specific to Illumina data; for additional information, please see the corresponding " +
             "<a href='https://www.broadinstitute.org/gatk/guide/article?id=6329'>GATK Dictionary entry</a>. </p>" +
+            "" +
+            "Note: For all Picard tools, headers that refer to PCT in the output metrics files, are referring to a fraction, not percentage." +
```

Comment:
Although now that I think about it, being *that* explicit in every tool doc is going to come off as a bit excessive. Maybe we can use this full-out version at the top of the Metrics page (which we need to redo because I forgot that that page is entirely auto-generated, as I just [commented here](https://github.com/broadinstitute/picard/commit/5b225242f0465252b28606bdb2bd6c298208bc34)). And on the metric-collecting tool pages, reduce it to a really short line:

"Note: Metrics labeled as percentages are actually expressed as fractions!"